src/common/utils/functor.ts

```diff
@@ -1,8 +1,15 @@
 /* eslint-disable @typescript-eslint/no-explicit-any */
+
+/**
+  @deprecated Not Used
+*/
 export const functor = <T extends Array<any>, U>(fn: any) => {
```

Comment:
I think so, yeah... Didn't do it because I wanted to ask you about them.
can you have a look locally and see if it's easy to remove that `agnosticFunctor` too? 
We can have a call later when you're finishing the review

Reply:
nice one, 
merged!